charts/trusted-artifact-signer/templates/nigthlymetrics.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ .Values.configs.nightlymetrics.name }}
+spec:
+  schedule: "00 00 * * *"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+          - name: test
+            image: "{{ .Values.configs.nightlymetrics.image.registry }}/{{ .Values.configs.nightlymetrics.image.repository }}:{{ .Values.configs.nightlymetrics.image.version }}"
+            imagePullPolicy: IfNotPresent
+            command:
+            - /bin/sh
+            - -c
+            - script.sh
+            - python main.py
```

Comment:
At the end of `script.sh` it calls `python3 main.py` so you can remove that. We also need to use a service account I think so that the `script.sh` has the proper permissions to get secrets from the cluster